az-digital/az_quickstart
# Fixes #1725 Disable Card Preview Links.

This PR makes card link previews not clickable. This is done by providing a new field formatter setting used by the card preview that avoids the default link behavior. The default card display has this setting turned off. The preview card setting has new configuration provided to provide its field formatter settings. An unneeded placeholder configuration setting is also removed.

This particular path (editing the preview display mode) was chosen because it involves no extra hooks and uses the built-in display system.

## Related issues
#1725 
Closes #1346
## How to test

- Create cards with links (esp clickable cards as these previews are most readily clickable)
- Verify links work on view display
- Verify links do not work in preview mode on edit page
- Verify pointer turns to a 'no' symbol on the edit page.
- Verify behavior can be successfully imported into new site
- Verify display settings page displays the correct "Interactive Links" setting for default and preview of the card paragraph.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
